python/META-INF/MANIFEST.MF

```diff
@@ -4,7 +4,7 @@ Bundle-Name: Python Plug-in for LibreOffice eclipse integration
 Bundle-SymbolicName: org.libreoffice.ide.eclipse.python;singleton:=true
 Bundle-Vendor: The Document Foundation
 Bundle-Version: 2.2.6
-Bundle-Activator: org.libreoffice.ide.eclipse.python.OOoJavaPlugin
+Bundle-Activator: org.libreoffice.ide.eclipse.python.OOoPythonPlugin
```

Comment:
OOoPythonPlugin(), start(BundleContext pContext) are called during the startup and they are called from the eclipse startup when the Eclipse is started with this particular plugin installed. Similarly stop(BundleContext pContext) is called from the Eclipse when the eclipse is closed.

As for the getDefault(), getImageDescriptor(String pPath and createImageRegistry() are called during New Project Wizard(s) specially for Java based, for example the Image [top right corner] which comes on the 2nd page of the New Extension Projet [Java][where we get the checkbox to include the test classes], is initialized from this place.